app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :user
+
+  after_save :update_comment_counter
+
+  def update_comment_counter
+    user.comments_count = user.comments.count
+    user.save
+    post.comments_count = post.comments.count
```

Comment:
I believe it is a better practice to check the count of comments from the actual `user.comments` that exist, rather than as a disconnected variable which could have bugs if the amount of comments changes separately from this method